models/value_set.go

```diff
@@ -49,61 +49,60 @@ type CodeSet struct {
 	Values []Concept
 }
 
-func (v ValueSetMap) CodeDisplayWithPreferredCodeForField(entry *Entry, coded *Coded, MapDataCriteria Mdc, codeType string, field string) CodeDisplay {
+func (v ValueSetMap) GenerateCodeDisplay(entry *Entry, coded *Coded, MapDataCriteria Mdc, codeType string) CodeDisplay {
 	codeDisplay, err := entry.GetCodeDisplay(codeType)
 	if err != nil {
-		log.Fatalln(err)
+		log.Fatal(err)
```

Comment:
I hate to bring this up in a PR that already does a good bit of refactoring, but can we figure out a way to get rid of the `log.Fatal`? They really need to be completely removed from this codebase very soon. A solution would be to change the function to return a pointer to a `CodeDisplay` so we can `return nil` and then the template won't execute because it was fed `nil`. This will of course result in a bad file, but at least it won't crash. We can even keep the log statement if we change it to a `log.Println`.